TicketBooking/Program.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace TicketBooking;
+
+internal class Program
+{
+    static void Main(string[] args)
+    {
+        UserInterface.UserInterface userInterface = new UserInterface.UserInterface();
+        userInterface.ShowInitialMenu();
+    }
```

Comment:
I would personally load the flights, users, etc from the .csv file on program start, and then pass the results to the repositories. I think this would be a simpler approach than having services / repositories have to load on usage.

@WaleedJubeh thoughts?

Reply:
Yes, I agree